.github/workflows/ci_workflow.yml

```diff
@@ -70,32 +70,159 @@ jobs:
       matrix:
         python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
 
+    container: python:${{ matrix.python-version }}-slim
+
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v4
-
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v5
-      with:
-        python-version: ${{ matrix.python-version }}
+      uses: actions/checkout@v5
 
-    - name: Install system dependencies
-      run: sudo apt-get update && sudo apt-get install -y graphviz openjdk-11-jre-headless
+    #- name: Set up Python ${{ matrix.python-version }}
+    #  uses: actions/setup-python@v5
+    #  with:
+    #    python-version: ${{ matrix.python-version }}
 
-    - name: Install Python dependencies (and EMMOntoPy)
+    - name: Install Python dependencies (and EMMOntoPy) minimal
       run: |
         python -m pip install --upgrade pip
         pip install -U setuptools wheel
         pip install -U -e .[dev]
```

Comment:
I name for dette steget står det at vi vill installerer EMMOntoPy minimal, men vi installerer dev - det er vel ikke minimal?

Reply:
Tja, det er minimal. Dev legger bare til pylint, pytest og pre-commit plus det som kreves for mkdocs-dokumentasjonen (ikke ontodoc reqs). Så det er minimal + det som trengs for å kjøre testene.